src/components/ModalContent/index.jsx

```diff
@@ -0,0 +1,49 @@
+// react library
+import React from 'react';
+
+import propTypes from 'prop-types';
+
+// styles
+import './ModalContent.scss';
+
+/**
+ * Component to render modal component
+ */
+export class ModalContent extends React.Component {
```

Comment:
Why is this a standalone component? And why is it a class? Please make it a functional component and also add it to the parent modal which should also be a functional component

Reply:
> Why is this a standalone component? And why is it a class? Please make it a functional component and also add it to the parent modal which should also be a functional component

Hello @iverenshaguy, thanks for the feedback.
The ModalContent should be a functional component since I removed all my state management from it. Thanks for that.
However, the modal its self should remain a class component because its state is crucial in my implementation logic.
Thanks, I will make changes and push.

Reply:
I wonder if you've heard of the React hook called `useState`? Kindly change the Modal component to a functional component. Also, this lack of knowledge of how React hook solves the problem using state in functional components makes me wonder if you were on the call during our live review with Ibukun.

Reply:
> I wonder if you've heard of the React hook called `useState`? Kindly change the Modal component to a functional component. Also, this lack of knowledge of how React hook solves the problem of using state in functional components makes me wonder if you were on the call during our live review with Ibukun.

I was on that call and this implementation was done before that call.
I just explained to you why I used class component and that is because I am managing state in my component. I know about the 'useState' method and am aware of its usage.
Please, are you saying that we should henceforth use functional components only?

Reply:
Were you really on the call? Because if you were, you would have heard when I said pages should be left as class components and other components should be functional components so that we can learn to use both. Also, your reply above suggested that it should be left as a class component because it uses state. Maybe you did not communicate what you meant to me properly. Please, update this component to a functional component.

If I may ask, why did you use a class component when you know the 'useState' **hook** and are aware of its usage?

Reply:
Thank you @iverenshaguy, I will update the component as requested.

Reply:
@iverenshaguy, if I get your right, you are also suggesting that I merge the Modal and ModalContent together right?
Because I separated them based on your request.